src/components/hooks/HookDetails.tsx

```diff
@@ -68,17 +60,59 @@ const useStyles = makeStyles(theme => {
 });
 
 interface Props {
-  hook: HookDetails_hook;
+  hook: HookDetails_hook$key;
 }
 
-function HookDetails(props: Props) {
-  let { hook } = props;
+export default function HookDetails(props: Props) {
+  let hook = useFragment(
+    graphql`
+      fragment HookDetails_hook on Hook {
+        id
+        repository {
+          ...RepositoryOwnerChip_repository
+          ...RepositoryNameChip_repository
+        }
+        build {
+          id
+          viewerPermission
+          ...BuildBranchNameChip_build
+          ...BuildChangeChip_build
+        }
+        task {
+          id
+          ...TaskNameChip_task
+        }
+        ...HookCreatedChip_hook
+        ...HookStatusChip_hook
+        name
+        info {
+          error
+          arguments
+          result
+          outputLogs
+          environment
+        }
+      }
+    `,
+    props.hook,
+  );
+
+  const [commitDetailsRerunMutation] = useMutation<HookDetailsRerunMutation>(graphql`
```

Comment:
Let's get the second `isInFlight` and disable ReRun button while a mutation is executing so people don't click multiple times.